chpl/chpl-service/src/main/java/gov/healthit/chpl/user/cognito/CognitoUserInvitation.java

```diff
@@ -0,0 +1,50 @@
+package gov.healthit.chpl.user.cognito;
+
+import java.util.Date;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class CognitoUserInvitation {
+    private Long id;
+    private String email;
+    private UUID invitationToken;
+
+    @JsonIgnore
+    private Date creationDate;
+
+    @JsonIgnore
+    private Date lastModifiedDate;
+
+    @JsonIgnore
+    private Long lastModifiedUserId;
+
+    @JsonIgnore
+    private UUID lasModifiedSsoUser;
+
+    public boolean isOlderThan(long numDays) {
+        return isOlderThanMillis(numDays * 24L * 60L * 60L * 1000L);
```

Comment:
Maybe consider moving the number to a constant since checkstyle is giving lots of notifications